.github/workflows/cypress.yml

```diff
@@ -133,7 +133,7 @@ jobs:
       matrix:
         node-version: [20]
         containers: [1, 2, 3]
-        php-versions: ['8.0', '8.2']
+        php-versions: ['8.0', '8.2', '8.3']
```

Comment:
@mejo- I think we currently don't conver the minimum supported php version anywhere. Any thought about running stable25 against 7.4 rather than 8.0? We could also run phpunit against all but i would pick the one that you feel has the better coverage for lines of code.

Reply:
No objections at all. We run the behat tests against PHP 7.4 on stable25 though, and my gut feeling is that we test the API endpoints pretty well with the behat tests. Not sure if Cypress covers more here.

Reply:
Thinking more about it, I'd also suggest to add PHP8.3 to the `behat-sqlite` matrix, given that these tests most likely have the highest coverage of PHP code.